packages/ui/src/lib/alert/ErrorMessage.svelte

```diff
@@ -1,16 +1,30 @@
 <script lang="ts">
 import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
+import { onMount } from 'svelte';
 import Fa from 'svelte-fa';
 
 import Tooltip from '../tooltip/Tooltip.svelte';
 
 export let error: string;
 export let icon = false;
+export let actionErrorInfo = false;
+let tipWidth: string | undefined = undefined;
```

Comment:
@benoitf can `tipWidth` being `undefined` cause problems when we use it in line 27 here or line 80 in `Tooltip.svelte`? I tried playing with it a bit and it doesn't look like it's problematic, but I just wanted to make sure. Is it better to leave it as undefined or maybe change it to start as an empty string?

Reply:
I think that now as you're using recommended `class={}` you should use as default an empty string and rename tipWidth to like class or customClass or something to say you provide custom class elements